Content.Server/GameObjects/Components/ConfigurationComponent.cs

```diff
@@ -29,12 +30,12 @@ namespace Content.Server.GameObjects.Components
     [ComponentReference(typeof(SharedConfigurationComponent))]
     public class ConfigurationComponent : SharedConfigurationComponent, IInteractUsing
     {
-        [ViewVariables] private BoundUserInterface UserInterface => Owner.GetUIOrNull(ConfigurationUiKey.Key);
+        [ViewVariables] private BoundUserInterface? UserInterface => Owner.GetUIOrNull(ConfigurationUiKey.Key);
 
         [ViewVariables]
         private readonly Dictionary<string, string> _config = new();
 
-        private Regex _validation;
+        private Regex _validation = new Regex("^[a-zA-Z0-9 ]*$", RegexOptions.Compiled);
```

Comment:
unneeded allocation as it gets overridden by exposedata

Reply:
mark it as nullable & initialize with null!

Reply:
😅